### PR TITLE
RRFS smoke update

### PR DESCRIPTION
**Pull request by cshartsough**
This is to update the algorithm (**gribdata**) for Wildfire Potential (firewx) and add it to the **image_list** for RRFS NA 3km runs.

Also made a change to the units in **default_specs**.

passed pylint and pytest.

Sample of latest image.
image

### CODE REVIEW

File: image_lists/rrfs_subset.yml [hunk 1]
```diff
@@ -92,7 +94,7 @@ hourly:
       - sfc
     ptmp:
       - 2m
-    ptyp
+    ptyp:
```
Comment (cshartsough) [inline image_lists/rrfs_subset.yml:97]: I noticed this typo while adding firewx to the file.

Comment (Brian-Jamison) [reply image_lists/rrfs_subset.yml:97]: I guess that was my fault.  Sorry.  Thanks for fixing it.

**Review by Brian-Jamison — APPROVED** [review]
Looks good!

**Review by christinaholtNOAA — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**